drupal/modules/lagoon/lagoon_search/lagoon_search.info.yml

```diff
@@ -1,7 +1,7 @@
 name: Lagoon search
 description: Provides integration between your Drupal site and Lagoon solr.
 package: Lagoon
-version: 8.x-1.00
+version: 8.x-1.01
```

Comment:
unless this is a patch-level change, in which case it would be `8.x-1.0-1`, this should be just `8.x-1.1`